test/index.test.mjs

```diff
@@ -1,17 +1,15 @@
-import expect from 'expect';
 import puppeteer from 'puppeteer';
 import helpers from '@percy/sdk-utils/test/helpers';
 import percySnapshot from '../index.js';
 
 describe('percySnapshot', () => {
-  let browser, page;
+  let browser, page, stdout, stderr;
```

Comment:
Are `stdout` and `stderr` needed anymore? I figured lint would fail if they weren't 🤔